README.md

```diff
@@ -29,14 +29,29 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your team lead
 
 1. Describe the biggest difference between `.forEach` & `.map`.
+   for forEach
+
+Answer: .forEach allows a callback function to mutate the current array while .map returns a new array.
```

Comment:
not quite. look into your use of mutate here and the different returns for .forEach and .map